schedule-pod-restart/templates/cronjob.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "schedule-pod-restart.name" . }}
```

I would need to check if this is valid, but if it is - Then it would be nice to add our typical labels as well.
```suggestion
  name: {{ include "schedule-pod-restart.name" . }}  
  labels:
    {{- include "schedule-pod-restart.labels" . | nindent 4 }}
```

---

Added in commit https://github.com/CoditEU/charts/pull/20/commits/807d90c57c6f579bacdf4b908af7e96a4a3a2d35 and tested (so syntax is correct)